app/utils/tests/sagaInjectors.test.js

```diff
@@ -24,7 +24,7 @@ describe('injectors', () => {
 
   describe('getInjectors', () => {
     beforeEach(() => {
-      store = configureStore({}, memoryHistory);
+      store = configureStore({}, createMemoryHistory());
```

Comment:
Is it possible to just run `createMemoryHistory` once before the tests and pass that each time?

Reply:
(Same thing for the other tests which use this function. It seems like an unnecessary performance overhead.)

Reply:
I just replaced what was already there to make it work. I will take a look at it again.

Reply:
Didn't know your requirements about running the clean script will do. I did npm run build and it builds and runs in production mode fine.